src/codegen/llvm.zig

```diff
@@ -1210,6 +1210,18 @@ pub const Object = struct {
         if (emit_asm_path == null and emit_bin_path == null and
             emit_llvm_ir_path == null and emit_llvm_bc_path == null) return;
 
+        if (emit_llvm_bc_path) |path| {
+            if (!self.builder.useLibLlvm()) {
+                var file = try std.fs.cwd().createFileZ(path, .{});
+                defer file.close();
+                const bitcode = try self.builder.toBitcode(arena);
+
+                const ptr: [*]const u8 = @ptrCast(bitcode.ptr);
+                try file.writeAll(ptr[0..(bitcode.len * 4)]);
+                return;
```

Comment:
```suggestion
                if (emit_asm_path == null and emit_bin_path == null and emit_llvm_ir_path == null) return;
```
It will eventually be the frontend's responsibility to check if the requested emits are invalid, but for now, not creating an expected output file and returning success here will just confuse the build system.

Reply:
Yeah I agree, that return was mostly for testing. I'll change it to handle it more reasonably.

Reply:
Yeah I figured, feel free to not resolve it until this is ready for review.